[2025-05-20T08:20:09.539+0000] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs scheduled__2025-05-19T00:00:00+00:00 [upstream_failed]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=1, upstream_failed=0, removed=0, done=1, success_setup=0, skipped_setup=0), upstream_task_ids={'fetch_transaction_data'}
[2025-05-20T08:20:09.540+0000] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs scheduled__2025-05-19T00:00:00+00:00 [upstream_failed]>, dependency 'Task Instance State' FAILED: Task is in the 'upstream_failed' state.
[2025-05-20T08:20:09.543+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
[2025-05-20T18:31:16.800+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs scheduled__2025-05-19T00:00:00+00:00 [queued]>
[2025-05-20T18:31:16.809+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs scheduled__2025-05-19T00:00:00+00:00 [queued]>
[2025-05-20T18:31:16.810+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-20T18:31:16.828+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): upload_to_hdfs> on 2025-05-19 00:00:00+00:00
[2025-05-20T18:31:16.842+0000] {standard_task_runner.py:57} INFO - Started process 450 to run task
[2025-05-20T18:31:16.846+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_transaction_data_and_upload_to_hdfs', 'upload_to_hdfs', 'scheduled__2025-05-19T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmp3es9ckmg']
[2025-05-20T18:31:16.850+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask upload_to_hdfs
[2025-05-20T18:31:16.908+0000] {task_command.py:415} INFO - Running <TaskInstance: fetch_transaction_data_and_upload_to_hdfs.upload_to_hdfs scheduled__2025-05-19T00:00:00+00:00 [running]> on host 903c87353ff8
[2025-05-20T18:31:17.201+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_transaction_data_and_upload_to_hdfs' AIRFLOW_CTX_TASK_ID='upload_to_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-19T00:00:00+00:00'
[2025-05-20T18:31:29.868+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-20T18:31:29.888+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fetch_transaction_data_and_upload_to_hdfs, task_id=upload_to_hdfs, execution_date=20250519T000000, start_date=20250520T183116, end_date=20250520T183129
[2025-05-20T18:31:29.955+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-20T18:31:29.994+0000] {taskinstance.py:2784} INFO - 2 downstream tasks scheduled from follow-on schedule check
